scss/_dropdown.scss

```diff
@@ -66,6 +66,18 @@
     left: 0;
     margin-top: var(--#{$prefix}dropdown-spacer);
   }
+
+  @if $dropdown-padding-y == 0 {
+    > .dropdown-item:first-child,
```

Comment:
Late to the party, but `.dropdown-menu > li` isn't right, there's usually an anchor (`<a>`) tag in between.
Shall we just skip the `>`?

Reply:
@mdo I'm sure you get auto-tagged, but just in case you don't.

Reply:
@mdo 
This still isn't working right, but I noticed I wrote it wrong. I meant that usually there's a `<li>` in between the `.dropdown-menu` and the `<a>`.

Taken from the Bootstrap example docs:
```html
  <ul class="dropdown-menu">
    <li><a class="dropdown-item" href="#">Menu item</a></li>
    <li><a class="dropdown-item" href="#">Menu item</a></li>
    <li><a class="dropdown-item" href="#">Menu item</a></li>
  </ul>
```

The selector right now is `.dropdown-menu > .dropdown-item`, but that won't work for the HTML above, as there's an `<li>` in between.